blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -0,0 +1,347 @@
+        )
+        : null}
+      {(promoElements.showHeadline)
+        ? (
+          <div className="results-list--headline-container mobile-order-1">
+            <a href={url} title={headlineText}>
+              <Heading className="headline-text">{headlineText}</Heading>
+            </a>
+          </div>
+        )
+        : null }
+      {(promoElements.showDescription || promoElements.showDate || promoElements.showByline)
+        ? (
+          <div className="results-list--description-author-container mobile-order-3">
+            {(promoElements.showDescription && descriptionText)
+              ? (
+                <a href={url} title={headlineText}>
+                  <SecondaryFont as="p" className="description-text">
+                    {descriptionText}
+                  </SecondaryFont>
+                </a>
+              )
+              : null}
+            {(promoElements.showDate || promoElements.showByline)
+              ? (
+                <div className="results-list--author-date">
+                  {(promoElements.showByline)
+                    ? (
+                      <Byline
+                        content={element}
+                        list
+                        separator={promoElements.showDate}
+                        font="Primary"
+                      />
+                    )
+                    : null }
+                  {(promoElements.showDate)
+                    ? <ArticleDate classNames="story-date" date={displayDate} />
+                    : null }
+                </div>
+              )
+              : null}
+          </div>
+        )
+        : null }
+    </div>
+  );
+}));
+
+const Results = ({
+  customFields,
+  fusionContext,
+  fusionProperties,
+  imageProperties,
+}) => {
+  const {
+    lazyLoad,
+    listContentConfig: {
+      contentService,
+      contentConfigValues,
+    },
+  } = customFields;
+
+  const {
+    arcSite,
+    contextPath,
+    deployment,
+    isAdmin,
+  } = fusionContext;
+
+  const {
+    fallbackImage,
+    locale,
+  } = fusionProperties;
+
+  const [configuredOffset] = useState(
+    parseInt(contentConfigValues.offset, 10)
+    || parseInt(contentConfigValues.feedOffset, 10)
+    || parseInt(contentConfigValues.from, 10)
+    || 0,
+  );
+  const [configuredSize] = useState(
+    parseInt(contentConfigValues.size, 10)
+    || parseInt(contentConfigValues.feedSize, 10)
+    || 10,
+  );
+
+  const [queryOffset, setQueryOffset] = useState(configuredOffset);
+
+  const [targetFallbackImage] = useState(
+    !(fallbackImage.includes('http'))
+      ? deployment(`${contextPath}/${fallbackImage}`)
+      : fallbackImage,
+  );
+
+  const placeholderResizedImageOptions = useContent({
+    source: !targetFallbackImage.includes('/resources/') ? 'resize-image-api' : null,
+    query: { raw_image_url: targetFallbackImage, respect_aspect_ratio: true },
+  });
+
+  const [isServerSideLazy] = useState(lazyLoad && isServerSide() && !isAdmin);
```

Comment:
this also seems like a candidate for refactoring. not sure why this is necessary to save in state. I'd recommend this blog post on when and when not to sync state https://kentcdodds.com/blog/dont-sync-state-derive-it

Reply:
```suggestion
  const isServerSideLazy= lazyLoad && isServerSide() && !isAdmin;
```

Reply:
if you look elsewhere in the code, this similar functionality is not set as react state

Reply:
Yep, see above.

Reply:
"Yep", you're saying you see this elsewhere in the code? 
![Screen Shot 2021-07-08 at 16 21 45](https://user-images.githubusercontent.com/5950956/124992399-81faf580-e008-11eb-995d-9f9c2d485b0a.png)

Reply:
sorry, "yep" I am removing the useState.  Sorry for the confusion.